package.json

```diff
@@ -43,7 +43,7 @@
     "@types/semver": "^6.2.0",
     "@typescript-eslint/parser": "^2.13.0",
     "@zeit/ncc": "^0.20.5",
-    "codecov": "^3.7.1",
+    "codecov": "^3.8.3",
```

Comment:
I upgraded to avoid codecove issues.
https://github.com/codecov/codecov-node/issues/284